### PR TITLE
Desc fix

### LINKED ISSUE #37302: Resonator Blast Mod Disk doesn't function
Issue reported from  Round ID: 86763 (/tg/Station Terry [EU] [100% LAG FREE])

Reporting client version: 512

The Resonator Blast Mod Disk (a design disk dropped from lavaland tendrils) says it's accepted by autolathes when you click on them, but the design is never added to the menu.

### PR DESCRIPTION
**Pull request by Jalleo**
Fixes #37302

### CODE REVIEW

**Comment by TribeOfBeavers** [comment]
This wasn't the issue. 

You literially get a message saying it has been accepted by the autolathe after a short channel but nothing changes. It should just refuse the disk.

Also the cargo protolathe outright refuses it so the description should say it goes in the research computer.

**Comment by Jalleo** [comment]
Right sorry I am not perfectly aware in this area of the code I shall correct tomorrow

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -79,7 +79,7 @@
 //KA modkit design discs
 /obj/item/disk/design_disk/modkit_disc
 	name = "KA Mod Disk"
-	desc = "A design disc containing the design for a unique kinetic accelerator modkit."
+	desc = "A design disc containing the design for a unique kinetic accelerator modkit. It's compatible with the research console"
```
Comment (SpaceManiac) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Missing terminal `.`.

Comment (Jalleo) [reply code/modules/mining/lavaland/necropolis_chests.dm]: Bah

**Pull request merged**